<launch>
    <node name="control_node" pkg="av_control" type="control_node" output="screen">
        <rosparam command="load" file="$(find av_control)/config/av_control.yaml" />
        <rosparam command="load" file="$(find av_control)/config/filter.yaml" />
    </node>
</launch>
